root/inc/favorite.tx

```diff
@@ -10,3 +10,11 @@
 <div class="show-logged-out">
     <button class="fav-not-logged-in favorite[% if $favorites { %] highlight[% } %]"><span>[% $favorites %]</span> ++</button>
 </div>
+%% if $stars {
```

Comment:
Should the control disappear when there are zero stars?

Reply:
That was my original thought, but now I'm not sure.